src/containers/DeleteConfirmation.jsx

```diff
@@ -0,0 +1,66 @@
+import styles from '../styles/deleteconfirmation'
+import classNames from 'classnames'
+
+import React from 'react'
+import { connect } from 'react-redux'
+import { translate } from '../lib/I18n'
+
+import { hideDeleteConfirmation, toggleFileSelection, trashFile } from '../actions'
+
+const DeleteConfirmation = ({ t, selected, onConfirm, onDismiss }) => (
+  <div className={styles['fil-deleteconfirmation']}>
+    <div className={styles['coz-overlay']}>
+      <div className={styles['coz-modal']}>
+        <h2 className={styles['coz-modal-title']}>
+          {t('deleteconfirmation.title')}
+        </h2>
+        <button
+          className={classNames('coz-btn', 'coz-btn--close', styles['coz-modal-close'])}
+          onClick={() => onDismiss(selected)}
+          >
+          <span className='coz-hidden'>{t('deleteconfirmation.close')}</span>
+        </button>
+        <p className={classNames(styles['fil-deleteconfirmation-text'], styles['icon-trash'])}>
+          {t('deleteconfirmation.trash')}
+        </p>
+        <p className={classNames(styles['fil-deleteconfirmation-text'], styles['icon-restore'])}>
+          {t('deleteconfirmation.restore')}
+        </p>
+        <p className={classNames(styles['fil-deleteconfirmation-text'], styles['icon-shared'])}>
+          {t('deleteconfirmation.shared')}
+        </p>
```

Comment:
If `className` changes, you change it once. If you had to change something, you change it once. DRY